.php_cs

```diff
@@ -1,12 +1,16 @@
 <?php
 
-$finder = Symfony\CS\Finder\DefaultFinder::create()
+$finder = PhpCsFixer\Finder::create()
     ->exclude('var')
-    ->name('console')
```

Comment:
Means it no longer checks the `bin/console` file?

Reply:
The finder find it by adding `name()`, but the fixer ignores it nonetheless. Investigating.